### PR TITLE
[#147] Implement Stringer interface and Parse method for common types

### PR DESCRIPTION
**Pull request by im-kulikov**
Implement Stringer interface and Parse method for well-known types

- [x] Owner package: ID
- [x] Object package: ID, Address
- [x] Container package: ID
- [x] Add constant error type

### CODE REVIEW

**Review by realloc — APPROVED** [review]
Commits may have had more unique summary, but LGTM =)

**Comment by im-kulikov** [comment]
> Commits may have had more unique summary, but LGTM =)

pls review changes

File: pkg/owner/id_test.go [hunk 1]
```diff
@@ -30,3 +32,33 @@ func TestNewIDFromNeo3Wallet(t *testing.T) {
 	id := NewIDFromNeo3Wallet(wallet)
 	require.Equal(t, id.ToV2().GetValue(), wallet.Bytes())
 }
+
+func TestID_Parse(t *testing.T) {
+	t.Run("should parse successful", func(t *testing.T) {
+		for i := 0; i < 10; i++ {
+			t.Run(strconv.Itoa(i), func(t *testing.T) {
+				wallet, err := NEO3WalletFromPublicKey(&test.DecodeKey(1).PublicKey)
```
Comment (alexvanin) [inline pkg/owner/id_test.go:40]: For cycle I suggest to decode different keys with `test.DecodeKey(i).PublicKey`

File: pkg/object/address.go [hunk 2]
```diff
@@ -59,3 +68,34 @@ func (a *Address) GetObjectID() *ID {
...
 	(*refs.Address)(a).SetObjectID(id.ToV2())
 }
+
+// Parse converts base58 string representation into Address.
+func (a *Address) Parse(s string) error {
+	var (
+		err   error
+		oid   = NewID()
+		cid   = container.NewID()
+		parts = strings.Split(s, "/")
```
Comment (alexvanin) [inline pkg/object/address.go:78]: Please add `const AddressSeparator = "/"` and use it there and in `String()`

**Review by alexvanin — APPROVED** [review]
Looks good. Proposed pair of small enhancements.

File: pkg/owner/id_test.go (hunk 1 above)
Comment (im-kulikov) [reply pkg/owner/id_test.go:40]: oh, you're right

File: pkg/object/address.go (hunk 2 above)
Comment (im-kulikov) [reply pkg/object/address.go:78]: yeah, it makes sense

**Comment by im-kulikov** [comment]
> Looks good. Proposed pair of small enhancements.

Thanks a lot, the proposed changes are really helpful!

File: pkg/object/address.go (hunk 2 above)
Comment (im-kulikov) [reply pkg/object/address.go:78]: decided to use `addressSeparator`

File: pkg/owner/id_test.go (hunk 1 above)
Comment (im-kulikov) [reply pkg/owner/id_test.go]: fixed

File: pkg/object/address.go (hunk 2 above)
Comment (im-kulikov) [reply pkg/object/address.go]: fixed

**Review by alexvanin — APPROVED** [review]

**Pull request merged**